### PR TITLE
ISSUE #1565: Update 'upgrade' doc for 4.8 release

### PR DESCRIPTION
**Pull request by reddycharan**
Descriptions of the changes in this PR:

Update 'upgrade' doc by adding an entry
"4.7.x to 4.8.0 upgrade" for ExplicitLac persistence

Master Issue: #1565

### CODE REVIEW

**Comment by reddycharan** [comment]
@sijie @eolivelli

File: site/docs/latest/admin/upgrade.md [hunk 1]
```diff
@@ -169,5 +169,8 @@ However, we list a list of things that you might want to know.
...
     to take a bookie out and recover it if you want to rollback to 4.4.x.
 
-If you are planning to upgrade a non-secured cluster to a secured cluster enabling security features in 4.5.0, please read [BookKeeper Security](../../security/overview) for more details.
+### 4.7.x to 4.8.X upgrade
+
+In 4.8.x new feature is added to persist explicitLac in FileInfo and explicitLac entry in Journal. Hence current journal format version is bumped to 6 and current FileInfo header version is bumped to 1. But since default config values of 'journalFormatVersionToWrite' and 'fileInfoFormatVersionToWrite' are set to older versions, this feature is off by default. To enable this feature those config values should be set to current versions. Once this is enabled then we cann't rollback to previous Bookie versions (4.7.x and older), since older version code won't be able to deal with explicitLac entry in Journal file while replaying journal and also reading Header of Index files / FileInfo would fail reading Index files with newer FileInfo version. So in summary, it is a non-rollbackable feature.
```
Comment (eolivelli) [inline site/docs/latest/admin/upgrade.md]: Typo: 'a new feature'

Typo: 'cann't'.

We can also state that the problem is present even if you are not using ExplicitLAC.

Overall is good

**Comment by reddycharan** [comment]
@eolivelli @sijie can you check the latest commit.

**Comment by reddycharan** [comment]
@sijie btw persisting explicitLac would work only for ILStorage and SLStorage but not for DBLStorage because of known issue - https://github.com/apache/bookkeeper/issues/1533. Should this be mentioned somewhere?

**Comment by sijie** [comment]
@reddycharan yeah. that should be mentioned in the doc. can you add that as well?

**Comment by reddycharan** [comment]
@sijie added note section

**Pull request closed without merging**